test/app-luatest/http_client_unit_test.lua

```diff
@@ -10,12 +10,10 @@ local ffi = require("ffi")
 
 local g = t.group('http_client_unit')
 
-package.loaded.http = {
-    client = {
-        new = function() return end
-    }
+package.loaded['http.client'] = {
+    new = function() return end
 }
-local driver = package.loaded.http.client -- luacheck: no unused
+local driver = package.loaded['http.client'] -- luacheck: no unused
```

Comment:
Well, now I see that tests are also adjusted. So, feel free to disregard [this review comment]().

Reply:
Okay.